provider/doc_edits.go

```diff
@@ -134,6 +135,24 @@ var fixUpEcsServiceNameForceNewDeployment = targetedSimpleReplace(
 		"(e.g., `myimage:latest`), roll Fargate tasks onto a newer platform version, or immediately deploy "+
 		"`ordered_placement_strategy` and `placement_constraints` updates.\n"+
 		"When using the forceNewDeployment property you also need to configure the triggers property.\n")
+var fixUpBucketReplicationConfig = targetedSimpleReplace(
+	"s3_bucket_replication_configuration.html.markdown",
```

Comment:
Could we put a check in (a test) that ensures this content is matching upstream, if not take from upstream content directly? This way if upstream edits it away we'll remember to update the copy.

Reply:
We have hard failure patterns during `make schema`; I'm happy to add those. we should add them for the other fixes as well. It may be noisy during upgrades; that's the tradeoff. I think you're right to request this though.

Reply:
Not sure what the "hard failure patterns" are but appreciated - the change looks like we add content we own where in fact this is a workaround to propagate upstream content, this postpones the issue of possible discrepancies to the future, and I must assume this page is high visibility since we are attempting to patch it. Thanks!

Reply:
I've added a new function that follows the [suggested pattern](https://github.com/pulumi/pulumi-terraform-bridge/blob/master/docs/guides/docs-edits.md#edit-rules).

I'm going to follow up with a PR that addresses this for other large replaces.